### PR TITLE
Update VS for Mac tutorials for VS4M release

### LINKED ISSUE #2355: Update VS for Mac tutorials for VS4M release
VS for Mac has been released. Update the VS4M tutorials to remove the "preview" language.

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #2355

This PR removes the "preview" language from the VS for Mac tutorials (and fixes a few other little things).

Bugzilla report on the broken UI (*It doesn't affect this PR tho*): https://bugzilla.xamarin.com/show_bug.cgi?id=56921

### CODE REVIEW

File: docs/core/tutorials/using-on-mac-vs-full-solution.md [hunk 1]
```diff
@@ -18,15 +18,15 @@ Visual Studio for Mac provides a full-featured Integrated Development Environmen
...
 
 > [!NOTE]
-> Visual Studio for Mac is preview software. As with all preview versions of Microsoft products, your feedback is highly valued. There are a two ways you can provide feedback to the development team on Visual Studio for Mac:
-> * In Visual Studio for Mac, select **Help > Report a Problem** from the menu or **Report a Problem** from the Welcome screen, which opens a window for filing a bug report.
-> * To make a suggestion, select **Help > Provide a Suggestion** from the menu or **Provide a Suggestion** from the Welcome screen, which takes you to the [Visual Studio for Mac UserVoice webpage](https://visualstudio.uservoice.com/forums/563332-visual-studio-for-mac).
+> Your feedback is highly valued. There are a two ways you can provide feedback to the development team on Visual Studio for Mac:
```
Comment (mairaw) [inline docs/core/tutorials/using-on-mac-vs-full-solution.md]: typo: a two ways

File: docs/core/tutorials/using-on-mac-vs-full-solution.md [hunk 2]
```diff
@@ -18,15 +18,15 @@ Visual Studio for Mac provides a full-featured Integrated Development Environmen
...
+> * In Visual Studio for Mac, select **Help** > **Report a Problem** from the menu or **Report a Problem** from the Welcome screen, which opens a window for filing a bug report. You can track your feedback in the [Developer Community](https://developercommunity.visualstudio.com/spaces/8/index.html) portal.
```
Comment (mairaw) [inline docs/core/tutorials/using-on-mac-vs-full-solution.md]: should the dev community link take you to https://developercommunity.visualstudio.com/spaces/41/index.html instead?

File: docs/core/tutorials/using-on-mac-vs-full-solution.md [hunk 3]
```diff
@@ -18,15 +18,15 @@ Visual Studio for Mac provides a full-featured Integrated Development Environmen
...
+> * To make a suggestion, select **Help** > **Provide a Suggestion** from the menu or **Provide a Suggestion** from the Welcome screen, which takes you to the [Visual Studio for Mac UserVoice webpage](https://visualstudio.uservoice.com/forums/563332-visual-studio-for-mac).
 
 ## Prerequisites
 
 [.NET Core and OpenSSL](https://www.microsoft.com/net/core#macos)
 
-For more information on prerequisites, see the [Prerequisites for .NET Core on Mac](../../core/macos-prerequisites.md).
+For more information on prerequisites, see the [Prerequisites for .NET Core on Mac](../../core/macos-prerequisites.md). [Release Notes](https://www.visualstudio.com/news/releasenotes/vs2017-mac-relnotes), [Platform Compatibility](https://www.visualstudio.com/productinfo/vs2017-compatibility-mac), and [System Requirements](https://www.visualstudio.com/productinfo/vs2017-system-requirements-mac) are also available at the Visual Studio website.
```
Comment (mairaw) [inline docs/core/tutorials/using-on-mac-vs-full-solution.md]: i had some comments about this section in the other PR. Not sure if I'd add release notes links, etc. on this topic.

File: docs/core/tutorials/using-on-mac-vs.md [hunk 4]
```diff
@@ -16,15 +16,15 @@ ms.assetid: 8902e849-dd17-42c0-8264-cc7ae3927a0c
 Visual Studio for Mac provides a full-featured Integrated Development Environment (IDE) for developing .NET Core applications. This topic walks you through building a simple console application using Visual Studio for Mac and .NET Core.
 
 > [!NOTE]
-> Visual Studio for Mac is preview software. As with all preview versions of Microsoft products, your feedback is highly valued. There are a two ways you can provide feedback to the development team on Visual Studio for Mac:
-> * In Visual Studio for Mac, select **Help > Report a Problem** from the menu or **Report a Problem** from the Welcome screen, which will open a window for filing a bug report.
-> * To make a suggestion, select **Help > Provide a Suggestion** from the menu or **Provide a Suggestion** from the Welcome screen, which will take you to the [Visual Studio for Mac UserVoice webpage](https://visualstudio.uservoice.com/forums/563332-visual-studio-for-mac).
+> Your feedback is highly valued. There are a two ways you can provide feedback to the development team on Visual Studio for Mac:
```
Comment (mairaw) [inline docs/core/tutorials/using-on-mac-vs.md]: same feedback comments as for the previous topic

**Review by mairaw — DISMISSED** [review]

File: docs/core/tutorials/using-on-mac-vs-full-solution.md (hunk 3 above)
Comment (GuardRex) [inline docs/core/tutorials/using-on-mac-vs-full-solution.md]: I address it in the other one, so I'll revert the change here.

File: docs/core/tutorials/using-on-mac-vs.md (hunk 4 above)
Comment (GuardRex) [inline docs/core/tutorials/using-on-mac-vs.md]: The *Getting Started* section should drop out with the other PR, so I'll leave this alone on this one.

**Comment by GuardRex** [comment]
@mairaw Changes made ...

We'll have to see if I create a good 'ole :boom: *merge conflict* :boom: between the two PR's. :smile:

**Comment by BillWagner** [comment]
@GuardRex Can you fix the merge conflicts here as well? Then, @mairaw and I will review the updates.

**Comment by GuardRex** [comment]
@BillWagner Done :+1:

**Review by rpetrusha — APPROVED** [review]
LGTM. You can merge this when you're ready.

**Comment by rpetrusha** [comment]
@GuardRex, I just approved this one. Feel free to merge it when you're ready to.

**Comment by GuardRex** [comment]
Thanks @rpetrusha!

@mairaw, I'm going to go ahead per our discussion to give myself a peaceful and restful vacation. 🏖

**Pull request merged**